scripts/monolith/enable-oie.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
+
+# export all environment vars from testenv
+set -o allexport
+source testenv
+set +o allexport
+
+yarn -s --cwd test/e2e ts-node ./support/monolith/enable-oie.ts
```

Comment:
nit: add package.json to support dir, so script can be accessed from a well named workspace